py/server/deephaven/table.py

```diff
@@ -2278,7 +2278,7 @@ def update_by(self, ops: Union[UpdateByOperation, List[UpdateByOperation]],
         except Exception as e:
             raise DHError(e, "table update-by operation failed.") from e
 
-    def slice(self, start: int, stop: int) -> Table:
+    def slice(self, first_position_inclusive: int, last_position_exclusive: int) -> Table:
```

Comment:
@wusteven815 noticed that the server wasn't using as descriptive of names as it could be and suggested this change. Only concern I have is that technically param names are part of the method contract, since someone could have been calling `table.slice(start=0, stop=100)` in their own project?

@jmao-denver what do you think revert or keep?

Reply:
I would revert.  This is a breaking change without enough upside.

@jmao-denver agree?

Reply:
I agree.